src/layout/index.tsx

```diff
@@ -51,30 +71,95 @@ const Layout: React.FC<LayoutProps> = ({
               </a>
             </Link>
           </h1>
-          <nav className={style.header__nav}>
-            <ul>
-              <li>
-                <Link href="/ansatte">
-                  <a>Alle varianter</a>
-                </Link>
-              </li>
-              <li>
-                <a href="https://jobs.variant.no" rel="noopener">
-                  Bli en variant
-                </a>
-              </li>
-              <li>
-                <a href="http://variant.blog" rel="noopener">
-                  Blogg
-                </a>
-              </li>
-              <li>
-                <a href="http://handbook.variant.no" rel="noopener">
-                  Håndbok
-                </a>
-              </li>
-            </ul>
-          </nav>
+
+          <span hidden id="menu-label">
+            Hovedmeny
+          </span>
+
+          <div ref={burgerRef}>
+            <button
+              className={style.container}
+              id="hamburger"
+              aria-labelledby="menu-label"
+              aria-expanded={clickActive ? true : false}
+              onClick={() =>
+                clickActive ? setClickActive(false) : setClickActive(true)
```

Comment:
This can simpler be expressed as `onClick={() => setClickActive(!clickActive)}` as you want to toggle clickActive between true/false (which is the same as negating the current value).

Reply:
```suggestion
                setClickActive(!clickActive)
```

Reply:
Yes, thats right! thanks for the input :D